azure-pipelines.yml

```diff
@@ -195,24 +195,43 @@ jobs:
       sudo cp -v ninja /usr/local/bin/
     displayName: 'Install Ninja'
   - script: |
-      npm install --prefix . v8-android@7.5.1
-      unxz node_modules/v8-android/dist/lib.unstripped/v8-android/7.5.1/libs.tar.xz
-      tar -xf node_modules/v8-android/dist/lib.unstripped/v8-android/7.5.1/libs.tar
-      mkdir Dependencies/v8
-      mkdir Dependencies/v8/include
-      mkdir Apps/Playground/Android/app/src/main/jniLibs
-      cp -r node_modules/v8-android/dist/include Dependencies/v8
-      cp -r arm64-v8a Apps/Playground/Android/app/src/main/jniLibs
-      cp -r armeabi-v7a Apps/Playground/Android/app/src/main/jniLibs
-      cp -r x86 Apps/Playground/Android/app/src/main/jniLibs
-      cp -r x86_64 Apps/Playground/Android/app/src/main/jniLibs
-    displayName: 'Install V8 npm'    
+      cd Apps/Playground/Android
+      npm install
```

Comment:
nit: Is it possible/advisable to get Gradle to do this part, too?

Reply:
I tried without success to do it with gradle using shell command. No idea why. I'm adding a issue: https://github.com/BabylonJS/BabylonNative/issues/197